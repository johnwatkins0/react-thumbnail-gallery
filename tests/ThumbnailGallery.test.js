import React from 'react';
import renderer from 'react-test-renderer';

import TestGallery from '../example/src/TestGallery';

test('ThumbnailGallery renders correctly.', () => {
  const tree = renderer.create(<TestGallery />).toJSON();

  expect(tree).toMatchSnapshot();
});
