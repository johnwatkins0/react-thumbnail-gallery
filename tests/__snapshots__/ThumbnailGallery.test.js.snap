// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThumbnailGallery renders correctly. 1`] = `
<div
  className="ThumbnailGallery sc-EHOje iWEkEs"
>
  <div
    className="sc-bdVaJa UcgSa"
  >
    <img
      alt="Blue"
      height={600}
      src="/react-thumbnail-gallery/example/assets/blue.png"
      width={1200}
    />
  </div>
  <div
    className="sc-htpNat dhFSqw"
  >
    <h1
      className="sc-bxivhb jxIFkG"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Blue",
        }
      }
    />
    <p
      className="sc-ifAKCX slpxh"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur maximus tristique tempor. Integer scelerisque libero eu libero maximus accumsan. Ut consectetur nec elit finibus pellentesque.",
        }
      }
    />
    <button
      className="sc-bwzfXH cUzJuN"
      onClick={[Function]}
    >
      <img
        alt="Blue"
        height={150}
        src="/react-thumbnail-gallery/example/assets/blue-thumb.png"
        width={150}
      />
    </button>
    <button
      className="sc-bwzfXH iWBcjN"
      onClick={[Function]}
    >
      <img
        alt="Brown"
        height={150}
        src="/react-thumbnail-gallery/example/assets/brown-thumb.png"
        width={150}
      />
    </button>
    <button
      className="sc-bwzfXH iWBcjN"
      onClick={[Function]}
    >
      <img
        alt="Green"
        height={150}
        src="/react-thumbnail-gallery/example/assets/green-thumb.png"
        width={150}
      />
    </button>
    <button
      className="sc-bwzfXH iWBcjN"
      onClick={[Function]}
    >
      <img
        alt="Orange"
        height={150}
        src="/react-thumbnail-gallery/example/assets/orange-thumb.png"
        width={150}
      />
    </button>
    <button
      className="sc-bwzfXH iWBcjN"
      onClick={[Function]}
    >
      <img
        alt="Purple"
        height={150}
        src="/react-thumbnail-gallery/example/assets/purple-thumb.png"
        width={150}
      />
    </button>
    <button
      className="sc-bwzfXH iWBcjN"
      onClick={[Function]}
    >
      <img
        alt="Red"
        height={150}
        src="/react-thumbnail-gallery/example/assets/red-thumb.png"
        width={150}
      />
    </button>
  </div>
</div>
`;
